cookbooks/bcpc/templates/default/haproxy.cfg.erb

```diff
@@ -65,3 +65,14 @@ option tcpka
   <%= "server #{server['hostname']} #{server['bcpc']['management']['ip']}:3306 check port 3307 inter 1s rise 1 fall 1 observe layer4 backup" %>
 <% end -%>
 <% end -%>
+
+<% if @httpfs_servers.length > 0 -%>
+listen httpfs <%="#{node[:bcpc][:floating][:vip]}"%>:14000
+  timeout client 24h
+  timeout server 24h
+  balance leastconn
+  option httpchk GET / 
+<% @httpfs_servers.each do |server| -%>
+  <%= "redirect code 301 prefix http://#{float_host(server['fqdn'])}:14000" %>
```

Comment:
We should have some [health checking](http://www.haproxy.com/doc/aloha/7.0/haproxy/healthchecks.html) defined here